packages/clerk-js/src/ui/components/OrganizationProfile/ActionConfirmationPage.tsx

```diff
@@ -141,10 +141,7 @@ const ActionConfirmationPage = withCardStateProvider((props: ActionConfirmationP
           <Text localizationKey={actionDescription} />
 
           <Form.ControlRow elementId={confirmationField.id}>
-            <Form.Control
-              {...confirmationField.props}
-              required
-            />
+            <Form.PlainInput {...confirmationField.props} />
```

Comment:
❓ Shouldn't this have the `isRequired` property?

Reply:
the hook above is defined as 
```tsx
  const confirmationField = useFormControl('deleteOrganizationConfirmation', '', {
...
    isRequired: true,
...
  });
```
so we are good